lib/matplotlib/tests/test_colorbar.py

```diff
@@ -570,3 +570,17 @@ def test_colorbar_label():
 
     cbar3 = fig.colorbar(im, orientation='horizontal', label='horizontal cbar')
     assert cbar3.ax.get_xlabel() == 'horizontal cbar'
+
+@image_comparison(
+    baseline_images=['colorbar_norms'], extensions=['png'])
```

Comment:
Image test looks reasonable (no need to *overthink* into avoiding them either); perhaps you can use style="default"?  (ticks inwards look weird to me now :-), and then you don't need to specify viridis below explicitly)